operator/BUILD.bazel

```diff
@@ -67,6 +67,7 @@ genrule(
         "bundle/manifests/forklift.konveyor.io_plans.yaml",
         "bundle/manifests/forklift.konveyor.io_providers.yaml",
         "bundle/manifests/forklift.konveyor.io_storagemaps.yaml",
+        "bundle/manifests/forklift.konveyor.io_ovirtimageiopopulators.yaml",
```

Comment:
I know this comes from the other repo but ovirtimageiopopulators is not the best name, how about ovirtpopulators?